Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -1847,12 +1847,11 @@ public void DefaultRepeatCountIsThree()
             Assert.Equal<int>(expectedRepeatCount, result);
             // Teardown
         }
-
-        [Fact]
+            [Fact]
         public void RepeatWillPerformActionTheDefaultNumberOfTimes()
         {
             // Fixture setup
-            Fixture sut = new Fixture();
+            IFixture sut = new Fixture();

```

Comment:
Why was this test changed to work on `IFixture`? Doesn't it work on `Fixture` any longer?